src/main/java/org/truffleruby/core/thread/ThreadNodes.java

```diff
@@ -374,6 +374,10 @@ private InterruptMode symbolToInterruptMode(RubyLanguage language, RubySymbol sy
 
         @Specialization
         protected RubyThread allocate(RubyClass rubyClass) {
+            if (getContext().getOptions().THREAD_WARN) {
+                getContext().getDefaultBacktraceFormatter().printBacktraceOnEnvStderr("thread:", this);
```

Comment:
Other cases like `--backtraces-raise` always have a space after the `:`, I think it would best to have it here too.
```
raise: spec/truffle/options/fixtures/raise_rescue.rb:10:in `some_method': error (RuntimeError)
	from spec/truffle/options/fixtures/raise_rescue.rb:15:in `<main>'
```